presets/4.3/tune/Tehllama_533_TinyTrainer_3in_3S_TUNE_Options.txt

```diff
@@ -0,0 +1,151 @@
+#$ TITLE: FlyFive33 3in TinyTrainer 3S Tune
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: TUNE
+#$ STATUS: EXPERIMENTAL
+#$ KEYWORDS: TinyTrainer, 533, Spec, Race, Llama
+#$ AUTHOR: Daniel Appel / Tehllama && Armando Gallegos / Mondo && Evan Turner / HeadsUp
+#$ DESCRIPTION: This is an adaptation of the 533 TinyTrainer tune, and expects bidirectional DShot ESCs at 48kHz PWM with MedHigh timing. 
+#$ DESCRIPTION: These fly best on 3S batteries in the 450-550mAh capacity range with 533 1404 motors.
+#$ DISCUSSION: https://github.com/betaflight/firmware-presets/pull/233
+
+#$ INCLUDE: presets/4.3/tune/defaults.txt
+# name: FIVE33
+
+# master
+
+set dshot_idle_value = 700
+# Fallback if Dynamic Idle not working or not selected
+set dshot_bidir = ON
+
+
+set small_angle = 180
+
+set simplified_gyro_filter_multiplier = 130
+
+profile 0
+# profile 0
+
+set iterm_relax = RPY
+set iterm_relax_cutoff = 33
+
+set throttle_boost = 8
+set p_pitch = 69
+set i_pitch = 80
+set d_pitch = 40
+set f_pitch = 144
+set p_roll = 63
+set i_roll = 72
+set d_roll = 38
+set f_roll = 132
+set p_yaw = 63
+set i_yaw = 72
+set f_yaw = 132
+set d_min_roll = 28
+set d_min_pitch = 29
+set d_max_advance = 0
+
+set thrust_linear = 10
+
+set simplified_i_gain = 65
+set simplified_d_gain = 95
+set simplified_pi_gain = 140
+set simplified_dmax_gain = 110
+set simplified_feedforward_gain = 110
+set simplified_pitch_d_gain = 90
+set simplified_pitch_pi_gain = 105
+simplified_tuning apply
+
+
+#$ OPTION BEGIN (CHECKED): Set Motor Poles = 12
+set motor_poles = 12
+
+#$ OPTION END
+
+#$ OPTION BEGIN (CHECKED): Set VBat Sag Compensation = 100
+set vbat_sag_compensation = 100
+
+#$ OPTION END
+
+
+
+#$ OPTION BEGIN (CHECKED): Apply Filters
+#$ INCLUDE: presets/4.3/filters/defaults.txt
+set gyro_lpf1_static_hz = 325
+set gyro_lpf2_static_hz = 650
+set dyn_notch_count = 2
+set dyn_notch_q = 350
+set dyn_notch_min_hz = 100
+set dyn_notch_max_hz = 650
+set gyro_lpf1_dyn_min_hz = 325
+set gyro_lpf1_dyn_max_hz = 650
+
+set rpm_filter_harmonics = 2
+set rpm_filter_min_hz = 150
+set rpm_filter_fade_range_hz = 75
+
+set yaw_lowpass_hz = 125
+
+profile 0
+set dterm_lpf1_dyn_min_hz = 97
+set dterm_lpf1_dyn_max_hz = 195
+set dterm_lpf1_dyn_expo = 8
+set dterm_lpf1_static_hz = 97
+set dterm_lpf2_static_hz = 195
+set simplified_dterm_filter_multiplier = 130
+
+#$ OPTION END
+
+#$ OPTION BEGIN (CHECKED): Apply Dynamic idle to all profiles
```

Comment:
Why do we need Dynamic idle in all profiles when this preset setting up everything else in just a single profile?

Reply:
Good catch, need to fix this.

Reply:
Removed